python3/HTSeq/scripts/count.py

```diff
@@ -277,6 +278,12 @@ def write_to_samout(r, assignment, samoutfile, template=None):
                     elif multimapped_mode == 'all':
                         for fsi in list(fs):
                             counts[fsi] += 1
+                    elif multimapped_mode == 'fraction':
+                        for fsi in list(fs):
+                            counts[fsi] += 1 / len(fs)
```

Comment:
Would be great to use ``1.0 / len(fs)``. It is clearer and works for Python2 too.

Reply:
Fixed.